openthread_border_router/DOCS.md

```diff
@@ -65,6 +65,7 @@ Add-on configuration:
 | otbr_log_level     | Set the log level of the OpenThread BorderRouter Agent     |
 | firewall           | Enable OpenThread Border Router firewall to block unnecessary traffic |
 | nat64              | Enable NAT64 to allow Thread devices accessing IPv4 addresses |
+| network_device     | IP address and port to connect to a network-based RCP - note you still need to set a dummy device, e.g., /dev/ttyS3 |
```

Comment:
While at it, I think we should add a note that this can cause problems as the RCP protocol is timing sensitive.

```suggestion
| network_device     | IP address and port to connect to a network-based RCP (1) |

(1) Note you still need to set a dummy device e.g. /dev/ttyS3. Also note that the RCP protocol is not designed to be transferred over an IP network: It is a timing sensitive protocol. You might experience Thread issues if your network link has excessive latencies. As Thread on its own is networking capable, it is recommended to run a Thread border router on the system the RCP radio is plugged in.
```